rosa.makefile

```diff
@@ -0,0 +1,49 @@
+CLUSTER_NAME?=sapeic
```

Comment:
Should we also add checks to ensure that required environment variables (e.g., ROSA_TOKEN, CLUSTER_NAME, CLUSTER_ADMIN_PASSWORD) are set before running commands like the checks done for ARO?

Reply:
yes, we should. I will merge this PR and create another one with requested functionality